### PR TITLE
Fix cant open current file for vim

### PR DESCRIPTION
**Pull request by takezyou**
The markdown currently open in vim could not be previewed, so it was fixed.

Please give me a review🙇‍♂️

### CODE REVIEW

File: autoload/preview_markdown.vim [hunk 1]
```diff
@@ -100,8 +101,7 @@ function! preview_markdown#preview() abort
       redraw
     endif
 
-    let s:preview_buf_nr = term_start(parser, opt)
-    call delete(tmp)
+    let s:preview_buf_nr = term_start(cmd, opt)
```
Comment (skanehira) [inline autoload/preview_markdown.vim:107]: If you include the file in the command, remove the following options as they are no longer needed

```
          \ 'in_io': 'file',
          \ 'in_name': tmp,
```

**Review by skanehira — CHANGES_REQUESTED** [review]
Thank you PR :)
I have a comment

File: autoload/preview_markdown.vim [hunk 2]
```diff
@@ -100,8 +101,7 @@ function! preview_markdown#preview() abort
       redraw
     endif
 
-    let s:preview_buf_nr = term_start(parser, opt)
-    call delete(tmp)
```
Comment (skanehira) [inline autoload/preview_markdown.vim:104]: tmp file have to removed

**Comment by takezyou** [comment]
Thank you comment!!
Fixed pointers☺️

File: autoload/preview_markdown.vim [hunk 3]
```diff
@@ -100,7 +100,8 @@ function! preview_markdown#preview() abort
     endif
 
     let s:preview_buf_nr = term_start(cmd, opt)
-
+	
+	sleep 1
```
Comment (skanehira) [inline autoload/preview_markdown.vim]: If need to sleep, use job exit callback is more better.

e.g

Add opt

```
'exit_cb': function('s:remove_tmp', [tmp]),
```

Add `s:remove_tmp` function.

```
function! s:remove_tmp(tmp, channel, msg) abort
  call delete(tmp)
endfunc

**Review by skanehira — CHANGES_REQUESTED** [review]
Thank you :)
I have a comment

File: autoload/preview_markdown.vim (hunk 3 above)
Comment (takezyou) [reply autoload/preview_markdown.vim]: Okay!!

**Comment by takezyou** [comment]
I fixed code.
Thank you review!

File: autoload/preview_markdown.vim [hunk 4]
```diff
@@ -56,14 +61,13 @@ function! preview_markdown#preview() abort
...
     let opt = {
-          \ 'in_io': 'file',
-          \ 'in_name': tmp,
           \ 'hidden': 1,
           \ 'curwin': 1,
           \ 'term_finish': 'open',
           \ 'term_kill': 'kill',
           \ 'term_name': 'PREVIEW',
           \ 'term_opencmd': is_vert ? 'vnew|b %d' : 'new|b %d',
+	      \ 'exit_cb': function('s:remove_tmp', [tmp]),
```
Comment (skanehira) [inline autoload/preview_markdown.vim]: Last one point.
I want the indent to look prettier.

**Review by skanehira — CHANGES_REQUESTED** [review]
LGTM.
I have last one point.

**Comment by takezyou** [comment]
Thank you for being so busy.
Fixed the indent!!

**Review by skanehira — APPROVED** [review]
**LGTM!**
Thank you :)

**Pull request merged**